bin/clever.js

```diff
@@ -12,6 +12,13 @@ if (process.argv.includes('--autocomplete-index')) {
   process.env.CLEVER_QUIET = '1';
 }
 
+// These need to be set before other stuff
+// Disable color globally if the user asked to
+if (process.argv.includes('--nocolor')) {
```

Comment:
I think that `no-color` will be better suited. Most of other available option uses a `-` to split words, as for example `--no-update-notifier`.

Reply:
It wasn't working as the parser was seeing 2 different options (if I remember weel). It's why I removed the middle '-'

Reply:
I looked at why I got this behavior, as sometimes the `-` was OK, but not with this option. After several attempts and further analysis, I found that the issue occurs only if an option starts with `--no-`.

For example:

```
$ clever --no-color
Unknown option: color
```

```
$ clever --color-no
Unknown option: color-no
```

For the same reason, the `--no-update-notifier` option can't be used:

```
$ clever logs --no-update-notifier
Unknown option: update-notifier
```

I thought it was a bug/feature of cliparse, but after more deep dive I discovered it's a feature of the `minimist` package used by cliparse. If an argument starts with `--no-` this part is not included in its name and its value is considered as false. You can check that in `minimist` [parsing tests](https://github.com/minimistjs/minimist/blob/main/test/parse.js) and [code](https://github.com/minimistjs/minimist/blob/fa422e82ce6d06278a515343670ebdc161417b90/index.js#L169).

So I pushed an update to deal with this. Now, there is a `--color` option, `true` by default. The user can also use `--no-color` as a shorter form.